lib/matplotlib/colors.py

```diff
@@ -1346,6 +1346,60 @@ def __call__(self, value, clip=None):
     def inverse(self, value):
         return value
 
+class Norm2d:
+    """
+    Normalize a list of two values corresponding to two 1D normalizers
+    """
+    def __init__(self, norm_instances=None):
+        """
+        Parameters
+        ----------
+        norm_instances :
+            A list of length two having instances of 1D normalizers
+        """
+        if norm_instances is None:
```

Comment:
How do you handle the use case `[BoundaryNorm(), None]` or `[None, BoundaryNorm]`?